felix.go

```diff
@@ -466,6 +466,12 @@ func servePrometheusMetrics(configParams *config.Config) {
 				prometheus.Unregister(prometheus.NewProcessCollector(os.Getpid(), ""))
 			}
 		}
+		gaugeHost := prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Makes sense, due to my current lack of in-depth knowledge in Prometheus GoLang client lib for now, just took the 1st function which seemed to work. Currently changing this